### DISCUSSION TITLE
source for the hat light lenses?

### DISCUSSION THREAD

**Discussion by ecto1a2003** [gpstar81/GPStar-proton-pack#242] [General]
im looking for the hat light lenses shown in the hats.md walkthrough. is there a source? google keeps showing me hats for the head instead of for led.

also is there a walktrhough for the barrel tip light?

**Comment by gpstar81** [comment]
Sorry no walkthrough yet for the barrel tip. I believe the hat lights Dustin used came from Amazon but modded.

**Comment by JMills75** [comment]
Gbfans.com sells a set of hat light lenses.  These are the same lenses used in FE.  
https://gbfans.com/shop/hat-lights/

**Comment by nomakewan** [comment]
In addition to the GBFans hatlights linked above, GPStar also sells hatlights now.

Kit: https://gpstartechnologies.com/products/hatlight-kit
Parts: https://gpstartechnologies.com/products/hatlights
